cmd/clusterawsadm/api/bootstrap/v1alpha1/types.go

```diff
@@ -108,6 +108,10 @@ type EKSConfig struct {
 	// Fargate controls the configuration of the AWS IAM role for
 	// used by EKS managed machine pools.
 	Fargate *AWSIAMRoleSpec `json:"fargate,omitempty"`
+	// KMSAliasPrefix is prefix to use to restrict permission to KMS keys to only those that have an alias
+	// name that is prefixed by this.
+	// Defaults to cluster-api-provider-aws-*
+	KMSAliasPrefix string `json:"kmsAliasPrefix,omitempty"`
```

Comment:
Should this be optional for existing configs to work?

Reply:
Its currently broken as the controller policy doesn't have the  permissions. In the release note I've added an action required release note to make people know they need to use an alias with a specific prefix.